server/seed.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const argv = require('yargs').argv;
```

Comment:
I know that Josh used this, in reality unless you fully understand how it works, you can simply define the number of entries and the filename without having this fallback option.